tests/integration/setup_environment.py

```diff
@@ -30,61 +38,60 @@ def find_free_port(start: int, end: int) -> int:
 class ADCMPostgresContainer(PostgresContainer):
     def __init__(self: Self, image: str, network: Network) -> None:
         super().__init__(image)
-        self.adcm_env_kwargs = {"STATISTICS_ENABLED": 0}
-        self.network = network
-
-    def setup_postgres(self: Self, username: str, password: str, adcm_db_name: str) -> None:
-        postgres_port = find_free_port(postgres_port_range[0], postgres_port_range[1])
-
-        self.adcm_env_kwargs = self.adcm_env_kwargs | {
-            "DB_HOST": f"{postgres_name}_{postgres_port}",
-            "DB_USER": db_user,
-            "DB_NAME": db_name,
-            "DB_PASS": db_password,
-            "DB_PORT": str(postgres_port),
-        }
-
-        self.with_name(f"{postgres_name}_{postgres_port}")
-        self.password = password
-        self.with_network(self.network)
-        self.with_bind_ports(postgres_port, postgres_port)
-
-        self.start()
-        wait_container_is_ready(self)
+        self.name = postgres_name
+        self.with_name(self.name)
+        self.with_network(network)
+
+    def execute_statement(self: Self, statement: str) -> None:
+        exit_code, out = self.exec(f'psql --username test --dbname test -c "{statement}"')
```

Comment:
Why `--username test --dbname test`? Didn't saw this user and db was created

Reply:
default values set in Postgres testcontainers class/image